app/controllers/api/v1/application_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Api
```

Comment:
Should this be capitalized, as it is acronym. Also it would `API` in rails `ActionController::API`. But for doing that we would need to add this file to config/initializers

```
ActiveSupport::Inflector.inflections do |inflect| 
  inflect.acronym 'API'
end
```

Reply:
Very useful to know that!! Thanks!!!